chia/protocols/shared_protocol.py

```diff
@@ -17,6 +17,7 @@
 # These are passed in as uint16 into the Handshake
 class Capability(IntEnum):
     BASE = 1  # Base capability just means it supports the chia protocol at mainnet
+    BLOB_API = 2
```

Comment:
I would call this something different since we don't need to use blobs directly to obtain the speedup